components/combobox/src/auro-combobox.js

```diff
@@ -368,17 +361,15 @@ export class AuroCombobox extends LitElement {
 
     if (this.menu.optionSelected) {
       // Get first option since combobox is single-select
-      const [selected] = this.menu.optionSelected;
+      const selected = this.menu.optionSelected;
 
-      if (!this.optionSelected || this.optionSelected[0] !== selected) {
-        // Store as array
-        this.optionSelected = [selected];
+      if (!this.optionSelected || this.optionSelected !== selected) {
+        this.optionSelected = selected;
       }
 
-      if (!this.value || this.value[0] !== selected.value) {
-        // Store as array
-        this.value = [selected.value];
-        // Menu already expects array
+      if (!this.value || this.value !== selected.value) {
+        this.value = selected.value;
+
         this.menu.value = this.value;
       }
```

Comment:
Doesn't this change break multi-select?

Reply:
No, combobox never supported multiselect